R/S3.R

```diff
@@ -1720,3 +1720,91 @@ pander.nls <- function(x, digits = panderOptions('digits'), show.convergence = F
                        digits = digits, show.convergence = show.convergence, ...)
 }
 
+
+
+#' Prints an ols object from rms package in Pandoc's markdown.
+#' @param x an ols object
+#' @param long if to print the correlation matrix of parameter estimates. default(\code{FALSE})
+#' @param coefs if to the table of model coefficients, standard errors, etc. default(\code{TRUE})
+#' @param digits passed to \code{format}. Can be a vector specifying values for each column (has to be the same length as number of columns).
+#' @param round passed to \code{round}. Can be a vector specifying values for each column (has to be the same length as number of columns). Values for non-numeric columns will be disregarded.
+#' @param ... optional parameters passed to raw \code{pandoc.table} function
+#' @export
+pander.ols <- function (x, long = FALSE, coefs = TRUE,
+                        digits = panderOptions("digits"), round = panderOptions("round"), ...) {
+    requireNamespace("rms", quietly = TRUE)
```

Comment:
Sure. This was just to agree on preferred output and then I would add tests and DESCRIPTION